iphone/Classes/TiUIListView.m

```diff
@@ -1815,7 +1815,24 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
         [self.proxy fireEvent:@"dragstart" withObject:nil withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
     }
 }
-
+-(void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset{
+    
+    if ([self.proxy _hasListeners:@"scrolling"]) {
+        NSMutableDictionary *eventArgs = [NSMutableDictionary dictionary];
+        NSString* swipeString = @"none";
+        if (velocity.y > 0){
+            swipeString = @"up";
+        }
+        if (velocity.y < 0){
+            swipeString = @"down";
+        }
+        if(swipeString !=(id)[NSNull null])
```

Comment:
It can never be null, because you set the default value to `none`. So please leave the default value to nil and only check `if (swipeString != nil) {`. Also note that the bracket goes into the same line.